packages/app/src/elements/field.ts

```diff
@@ -156,8 +156,9 @@ export class FieldElement extends LitElement {
                     : [await randomString(16, charSets.alphanum), await generatePassphrase()];
                 break;
             case FieldType.Url:
-                this._suggestions =
-                    !this._valueInput?.value && app.state.context.browser?.url ? [app.state.context.browser.url] : [];
+                const url = app.state.context.browser?.url;
+                const origin = url && new URL(url).origin;
+                this._suggestions = !this._valueInput?.value && url && origin ? [origin, url] : [];
```

Comment:
Offer the choice between just the origin or the fully qualified url including path